Ignore containers that are not part of the network

sykesm opened the pull request:
Add additional filtering to the e2e tests that examine the state of docker containers to include the test network ID. This prevents the tests from observing transient containers from other networks or development tools that break assumptions or assertions in the tests.

jyellick commented:
Signed-off-by: Jason Yellick <jyellick@us.ibm.com>

All go tests pass
All behave tests pass

sykesm committed:
Ignore containers that are not part of the network

Add additional filtering to the e2e tests that examine the state of
docker containers to include the test network ID. This prevents the
tests from observing transient containers from other networks or
development tools that break assumptions or assertions in the tests.

FAB-17641 #done

Signed-off-by: Matthew Sykes <sykesmat@us.ibm.com>
